pages/ox_core/Functions/server.mdx

```diff
@@ -432,4 +432,34 @@ Ox.GetLicenses()
 
 **Returns**
 
-- `OxLicense[]`
\ No newline at end of file
+- `OxLicense[]`
+
+## Ox.GetGroupActivePlayers
+
+Returns an array of all active players in the specified group.
+
+```lua
+Ox.GetGroupActivePlayers(groupName)
+```
+**Parameters**
+
+- groupName: `string`
+
+**Returns**
+
+- `number[]`
+
+## Ox.GetGroupActivePlayersByType
+
+Returns an array of all active players aggregated from groups matching the specified type.
+
+```lua
+Ox.GetGroupActivePlayers(type)
```

Comment:
```suggestion
Ox.GetGroupActivePlayers(groupType)
```
Please rename this to groupType since this is marked as lua type looks wierd on the docs page.